userguide/tn_vcenter.rst

```diff
@@ -228,9 +229,11 @@ shows the configuration screen for the new datastore:
    Configuring a VMFS Datastore
 
 
-Enter a name for the new datastore and the desired size. Enter the IP
-address of the %brand% system in the :guilabel:`Data Path IP` box.
-Select the desired :guilabel:`VMFS Version` from the drop-down. If a
+Enter a name for the new datastore and the desired size. Datastore names
+can only contain alphanumeric characters and only the :literal:`-` and
```

Comment:
This is somewhat oddly worded.  First it says "can only contain alphumeric" and then "and only the - and ."